File: component/component-kaiheila-parent/component-kaiheila-v3/src/test/java/love/forte/test/Application.java
package love.forte.test;

import love.forte.simbot.annotation.SimbotApplication;
import love.forte.simbot.core.SimbotApp;
import love.forte.simbot.core.SimbotContext;

/**
 * @author ForteScarlet
 */
@SimbotApplication
public class Application {
    public static void main(String[] args) {
        SimbotContext run = SimbotApp.run(Application.class, args);

        System.out.println("Join and wait.");
        run.join();

    }
}

File: simbot-project-tests/simbot-project-test-boot/src/main/kotlin/love/forliy/test/boot/Application.kt
/*
 * Copyright (c) 2022-2023 ForteScarlet.
 *
 * This file is part of Simple Robot.
 *
 * Simple Robot is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
 *
 * Simple Robot is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public License along with Simple Robot. If not, see <https://www.gnu.org/licenses/>.
 */

package love.forliy.test.boot

import love.forte.simboot.core.SimbootApplication
import love.forte.simboot.core.simbootApp


/**
 *
 * @author ForteScarlet
 */
@SimbootApplication
class Application

suspend fun main(vararg args: String) {
    val app = simbootApp<Application>(args = args)
    app.join()
}
